lightspeed/flame
# Add Interac icon

## Description

Add Interac Icon

## How to test?

- Checkout branch, run `yarn dev`
- Open [Storybook](http://localhost:6006/?path=/story/icon--story)
- Or check the deploy preview on Netlify (link available in comments)

## Checklist
